daemon/src/git_helper.rs

```diff
@@ -0,0 +1,69 @@
+//! git-remote-rad git helper related functionality.
```

Comment:
Also doesn't seem to belong in the context of the daemon, either general utility pulled in from this repo or back in Upstream.